applications/nrf5340_audio/src/audio/CMakeLists.txt

```diff
@@ -14,3 +14,8 @@ if (CONFIG_BT_BAP_BROADCAST_SOURCE)
 	target_sources(app PRIVATE
 	       ${CMAKE_CURRENT_SOURCE_DIR}/streamctrl_broadcast_source.c)
 endif()
+
+if (CONFIG_BT_BAP_UNICAST_SERVER)
+	target_sources(app PRIVATE
+	       ${CMAKE_CURRENT_SOURCE_DIR}/streamctrl_unicast_server.c)
+endif()
```

Comment:
Need a newline in here

Reply:
There is a newline here. I think the GitHub viewer is wrong.

Reply:
There is something strange here. I have added an extra newline to see if that passes.